src/main/java/de/intension/keycloak/TwoStepLoginWithCodeAuthenticatorFactory.java

```diff
@@ -0,0 +1,107 @@
+package de.intension.keycloak;
+
+import java.util.List;
+
+import org.keycloak.Config;
+import org.keycloak.authentication.Authenticator;
+import org.keycloak.authentication.AuthenticatorFactory;
+import org.keycloak.models.AuthenticationExecutionModel;
+import org.keycloak.models.KeycloakSession;
+import org.keycloak.models.KeycloakSessionFactory;
+import org.keycloak.provider.ProviderConfigProperty;
+
+/**
+ * @author markus.haller
```

Comment:
No code ownership at intension :wink: remove and update Eclipse settings `Window > Preferences`
![Screenshot from 2021-06-14 11-01-29](https://user-images.githubusercontent.com/42644709/121867179-effd1700-ccff-11eb-9f3b-5419abbe3490.png)